app/src/main/kotlin/org/fossify/notes/dialogs/NewChecklistItemDialog.kt

```diff
@@ -67,6 +67,21 @@ class NewChecklistItemDialog(val activity: Activity, callback: (titles: ArrayLis
                     false
                 }
             }
+            titleEditText.onTextChangeListener { text ->
+                val lines = text.split("\n").filter { it.trim().isNotEmpty() }
```

Comment:
We can use `lines()` extension here:

```suggestion
                val lines = text.lines().filter { it.trim().isNotEmpty() }
```